src/index.js

```diff
@@ -34,6 +34,11 @@ export default class Marker extends Component {
     latLngToPixel: PropTypes.func,
     pixelToLatLng: PropTypes.func
   }
+  static defaultProps = {
+    markerSize: '30',
```

Comment:
The markerSize default prop is a `"string"`, but the PropTypes say it should be a `PropTypes.number`

Reply:
Fixed that